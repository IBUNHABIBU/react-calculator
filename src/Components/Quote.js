import React from 'react';

const Quote = () => (
  <div>
    <h1>Quote</h1>
    <p>Fashion is what seems beautiful now but looks ugly later</p>
  </div>
);

export default Quote;
