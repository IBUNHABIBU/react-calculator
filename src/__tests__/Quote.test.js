import { shallow } from 'enzyme';
import Quote from '../Components/Quote';

let wrapper;
beforeEach(() => {
  wrapper = shallow(<Quote />);
});

describe('Show the quote of the day', () => {
  it('should display the Quote heading', () => {
    expect(wrapper.find('h1').text()).toContain('Quote');
  });
});
