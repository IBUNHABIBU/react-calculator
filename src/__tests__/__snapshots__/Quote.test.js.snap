// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots renders App component correctly 1`] = `
<div>
  <h1>
    Quote
  </h1>
  <p>
    Fashion is what seems beautiful now but looks ugly later
  </p>
</div>
`;
